twilio_integration/twilio_integration/doctype/whatsapp_message/whatsapp_message.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2021, Frappe and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe.model.document import Document
+from six import string_types
+from frappe.utils.password import get_decrypted_password
+from frappe import _
+from twilio_integration.twilio_integration.doctype.twilio_settings.twilio_settings import get_twilio_client
+
+class WhatsAppMessage(Document):
+	def send(self):
+		client = get_twilio_client()
```

Comment:
Twilio_handler module has a class name Twilio, that handles all twilio connections. Can we move this over there?